### PR TITLE
Add three new make clean* targets to clean up box, lstmf, output.

### PR DESCRIPTION
**Pull request by songzy12**
This pull request would fix the feature request issue #89 .

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -328,8 +328,17 @@ tesseract-langs: $(TESSDATA)/eng.traineddata
...
-clean:
+# Clean generated .box files
+clean-box:
 	find $(GROUND_TRUTH_DIR) -name '*.box' -delete
+
+# Clean generated .lstmf files
+clean-lstmf:
 	find $(GROUND_TRUTH_DIR) -name '*.lstmf' -delete
+
+# Clean generated ouptut files
```
Comment (stweil) [inline Makefile:339]:
```suggestion
# Clean generated output files
```

File: Makefile [hunk 2]
```diff
@@ -328,8 +328,17 @@ tesseract-langs: $(TESSDATA)/eng.traineddata
...
+clean-output:
```
Comment (stweil) [inline Makefile:340]:
```suggestion
.PHONY: clean-output
clean-output:
```

File: Makefile [hunk 3]
```diff
@@ -328,8 +328,17 @@ tesseract-langs: $(TESSDATA)/eng.traineddata
...
 	cd $(TESSDATA) && wget https://github.com/tesseract-ocr/tessdata$(TESSDATA_REPO)/raw/master/$(notdir $@)
 
-# Clean all generated files
-clean:
+# Clean generated .box files
+clean-box:
 	find $(GROUND_TRUTH_DIR) -name '*.box' -delete
+
+# Clean generated .lstmf files
+clean-lstmf:
```
Comment (stweil) [inline Makefile:336]:
```suggestion
.PHONY: clean-lstmf
clean-lstmf:
```

File: Makefile [hunk 4]
```diff
@@ -328,8 +328,17 @@ tesseract-langs: $(TESSDATA)/eng.traineddata
 $(TESSDATA)/eng.traineddata:
 	cd $(TESSDATA) && wget https://github.com/tesseract-ocr/tessdata$(TESSDATA_REPO)/raw/master/$(notdir $@)
 
-# Clean all generated files
-clean:
+# Clean generated .box files
+clean-box:
```
Comment (stweil) [inline Makefile:332]:
```suggestion
.PHONY: clean-box
clean-box:
```

**Comment by stweil** [comment]
Thank you!

**Comment by songzy12** [comment]
Thanks for the code review!

**Pull request merged**